Monika After Story/game/script-topics.rpy

```diff
@@ -8960,6 +8960,245 @@ label monika_hydration:
+    m 1hublb "Ehehe~"
+    return
+
+define persistent._mas_didplayerbetray = None
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_betrayal",category=['misc'],prompt="Betrayal",random=True))
+
+label monika_betrayal:
+    m 1eka "...Say, [player]?"
+    m 1lka "Something came to my mind just now."
+    m 1eka "Have you ever betrayed someone by any means?"
+    m 4eka "I don't mean to 'betray' little promises with friends that could even be considered jokes."
+    m 4eku "I mean it in a real way... You know, breaking the trust someone had with you because you did something terrible to them."
+    m "You don't have to answer if you don't want to. It's completely understandable."
+    $ _history_list.pop()
+    menu:
+        m "You don't have to answer if you don't want to, it's completely understandable."
+        "Yes, I have betrayed someone.":
+            $ persistent._mas_didplayerbetray = True
+            m 2ekc "..."
+            m "It's alright, [player]."
+            m 3ekc "I know you probably didn't mean to do that."
+            m 3ekb "We all mess up sometimes, don't we?"
+            m "Maybe you didn't do it on purpose."
+            m "Or...maybe you did."
+            m 3ekc "Whatever happened, you have to make sure to never to that again, alright?"
+            m "You have to promise me."
+            m "Trust is not something to play with."
+            m "Try to keep that in mind."
+
+        "No, I haven't betrayed anyone.":
+            $ persistent._mas_didplayerbetray = False
+            m 2huu "I'm so happy to hear that, [player]."
+            m 3efb "People's trust isn't something to be played with."
+            m "So I'm glad to hear that from you."
+            m 3esa "And just for letting you know..."
+            m 3hublb "I...{w=1}will never betray you!~"
+
+        "...":
+            m 2ekc "It's alright, [player]."
+            m "I won't force you to say anything."
+            m "I understand it's a deep topic to talk about, so I won't push you out of respect."
+            m 3ekc "Besides, what kind of girlfriend would I be if I did that?"
+            m 3ekb "Let's just try to forget this conversation existed in first place."
+    return "derandom"
+        
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_photography",category=['misc'],prompt="Photography",random=True))
+
+label monika_photography:
+    m 1eua "...Say, [player]?"
+    m 1esa "Do you happen to know about photography?"
+    m 3esu "It has become an interesting topic for me."
+    m "The more I research it, the more I like it."
+    m 3lsu "For example, did you know that the first photograph was taken using a box with a hole in it as a camera?"
+    m "Lenses weren't introduced until much later."
+    m 3esb "Also, the firsts ways of photography reveal relied on a series of special liquids to do the task."
```

Comment:
I would replace the line with "Also, early photography relied on a series of special liquids to work."

Reply:
Got it! Thanks for the suggestion